www-apps/gitea/gitea-1.8.2.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit golang-vcs-snapshot systemd user
+
+EGO_PN="code.gitea.io/gitea"
+KEYWORDS="~amd64 ~arm ~arm64"
+
+DESCRIPTION="A painless self-hosted Git service"
+HOMEPAGE="https://gitea.io"
+SRC_URI="https://github.com/go-gitea/gitea/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="pam sqlite"
+
+COMMON_DEPEND="pam? ( sys-libs/pam )"
+DEPEND="${COMMON_DEPEND}
+	dev-go/go-bindata"
+RDEPEND="${COMMON_DEPEND}
+	dev-vcs/git"
+
+DOCS=( custom/conf/app.ini.sample CONTRIBUTING.md README.md )
+S="${WORKDIR}/${P}/src/${EGO_PN}"
+
+pkg_setup() {
+	enewgroup git
+	enewuser git -1 /bin/bash /var/lib/gitea git
```

Comment:
This is going to cause trouble in the future. I'm pretty sure we have other packages creating git user with different homedir.

Is the homedir strictly necessary? And do you really need to specify a regular shell for the user?

Reply:
`git` user -> yes, there other alternative packages using/creating a git-user.
however, gitea is a kind of gitlab/github equivalent - so cloning via ssh **git@HOSTNAME** is defacto standard, isn't it?

homedir is necessary for providing a place for the `.ssh/authorized_keys` file

shell -> again ssh access to execute `command="/usr/bin/gitea.."`

Reply:
Well, we'll probably end up by having `acct-user/git` and some USE-flags to control this then. But that's for later.